request_logging/middleware.py

```diff
@@ -141,25 +122,42 @@ def _should_log_route(self, request):
         no_logging = getattr(func, NO_LOGGING_ATTR, None)
         return no_logging
 
-    def _log_request_headers(self, request, no_logging, logging_context):
+    def _skip_logging_request(self, request, reason):
+        method_path = "{} {}".format(request.method, request.get_full_path())
+        no_log_context = {
+            'args': (),
+            'kwargs': {
+                'extra': {
+                    'no_logging': reason
+                },
+            },
+        }
+        self.logger.log(logging.INFO, method_path + " (not logged because '" + reason + "')", no_log_context)
```

Comment:
So the problem here is the logger isn't actually a logger but a strategy for injecting color into the outgoing log entry.   Your suggest will result in a symbol error since `self.logger` has no method named `info`.

Reply:
Ah, got it. I saw `logger` and assumed `logging`. Thanks.